packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -232,7 +232,19 @@ class ResidentWebRunner extends ResidentRunner {
       unawaited(_vmService.registerService('reloadSources', 'FlutterTools'));
       websocketUri = Uri.parse(_connectionResult.debugConnection.uri);
       // Always run main after connecting because start paused doesn't work yet.
-      _connectionResult.appConnection.runMain();
+      if (!debuggingOptions.startPaused) {
+        _connectionResult.appConnection.runMain();
+      } else {
+        // I'm not sure if this is necessary.
```

Comment:
If we take this away, won't main never be called?

Reply:
Yes, removed comment